bot/github.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const https = require('https');
+const config = require('../config');
+
+// Returns the url for getting the labels of a request (Github API v3)
+// req has req.issue.labels_url
+const getLabelsURL = function (req) {
+    let url = req.body.issue.labels_url;
+    // Make the URL generic removing the name of the label
+    return url.replace('{/name}', '');
+}
+
+// Returns  the bounty labelNames of the request, only for testing motives
+const getLabelsMock = function (req) {
+    return new Promise((resolve, reject) => { resolve(req.body.issue.labels) });
+}
+
+// Returns all the bounty labelNames of a given issue (Github API v3)
+const getLabels = function (req) {
+    if (config.debug) {
+        return getLabelsMock(req);
+    } else {
+        const path = getLabelsURL(req).replace('https://api.github.com', '');
+        const options = {
+            hostname: 'api.github.com',
+            path: path,
+            headers: { 'User-Agent': 'kafkasl' }
```

Comment:
Would this not need to be changed for different organizations?

Reply:
@andytudhope 'User-agent' is obtained from the configuration file (it was hardcoded in this version I am pushing the update with the fix) the hostname is common (for all using github as end-point).